language-service/src/services/yamlCompletion.ts

```diff
@@ -427,7 +441,7 @@ export class YAMLCompletion {
                     insertText: this.getInsertTextForValue(enm, separatorAfter),
                     insertTextFormat: InsertTextFormat.Snippet,
                     documentation
-                });
+                }, false);
```

Comment:
It looks like we need to check ignoreCase here as well. Example from the schema:
```
 "errorActionPreference": {
          "oneOf": [
            {
              "enum": [
                "stop",
                "continue",
                "silentlyContinue"
              ],
              "ignoreCase": "value"
            },
```

Reply:
Again, since this is a value completion, I don't think we need to check for case-insensitive conflicts